pegasus/sites.v3/code.org/public/css/page/poetry-styles.scss

```diff
@@ -0,0 +1,55 @@
+// Custom styles for code.org/poetry
+// — supplemental to shared/css/design-system-pegasus.scss
+
+@import 'breakpoints';
+
+// Hero banner
+.hero-banner-basic {
+  background: url('/images/banners/banner-naipi-pattern.png') center center;
+  background-color: #6C468A;
+
+  .text-wrapper {
+    width: 50%;
+
+    @media screen and (max-width: $width-sm) {
+      width: 100%;
+    }
+  }
+
+  figure {
+    margin-top: 3em;
+    width: 40%;
+    text-align: center;
+
+    @media screen and (max-width: $width-sm) {
+      width: 100%;
+      margin-top: 2rem;
+    }
+
+    .videothumbnail {
+      height: 250px;
+      width: auto;
+    }
+  }
+}
+
+section.bg-pattern-dark {
```

Comment:
Instead of adding styles here, you can add a new class (and background property) that's nested within the existing `.bg-pattern-dark` class on `design-system-pegasus.scss` (it would go on line 54). 

https://github.com/code-dot-org/code-dot-org/blob/1f8e4bc9e5369fb52f7922d8e66eb8fa20a43ef3/pegasus/sites.v3/code.org/public/css/design-system-pegasus.scss#L48-L53

This way we can use a new `.bg-pattern-dark.maze-dots` class in the markup in this PR and on other sections in the future.

```
section.bg-pattern-dark {
  background: url("/images/banners/banner-bg-lines-neutral-dark.png") center
    repeat;
  background-size: 12rem;
  background-color: var(--neutral_dark);

*** Add this on line 54 on design-system-pegasus.scss ***
  &.maze-dots {
    background: url("/images/banners/banner-bg-maze-dots-light.png") center
    repeat;
  }
}
```

Let me know if this is confusing and you want to pair!

Reply:
**Update:** Ignore all of the above. Moshe decided we can use an existing background pattern for the Featured Projects section. I'll comment in the code where to make this update. 

You can delete lines `37-41` here.